zerver/lib/exceptions.py

```diff
@@ -271,6 +272,17 @@ def msg_format() -> str:
         return _("Must be an organization or stream administrator")
 
 
+class UserDeactivatedError(JsonableError):
+    code: ErrorCode = ErrorCode.USER_DEACTIVATED
```

Comment:
I think for these we should probably set `http_status_code=403` -- @mateuszmandera do you agree?

Reply:
I have made `htttp_status_code=403` for now. I'll update it if required.

Reply:
Yeah 403 looks right to me here